graphql/schemabuilder/pagination.go

```diff
@@ -99,6 +100,10 @@ type ConnectionArgs struct {
 	Args interface{}
 	// filter: "text search"
 	Filter *string
+	// sortBy: "fieldName"
+	SortBy *string
```

Comment:
can we use an enum for these based on the keys in the map?

Reply:
maybe enum makes the embedding thing hard

Reply:
Ehhhhhhhhhhhhhh we can ~dynamically~ do it maybe

Reply:
Maybe this could be a phase 2?

Reply:
i think this would be enough of a win (with autocompletion, validation) to do now. if the tradeoff is against embeddable (passthrough) sorts, I think I'd rather have the enum support over the passthrough api.